content/docs/guides/saml/aad.md

```diff
@@ -38,22 +42,23 @@ The first step is to add a new Pulumi application to your Azure AD tenant.
 
     ![SAML configuration](/images/docs/reference/service/saml-aad/saml-configuration.png)
 
-The next step is to enter Pulumi-specific configuration data into your Azure AD application.
+### Enter Pulumi configuration into your Azure AD application
+
 The values you will want to use are dependent upon your organization's name on Pulumi. Be sure
 to replace `acmecorp` with whatever your specific Pulumi organization's name is.
 
 | SAML Setting | Value |
 | --------------- | ----- |
-| Identifier (Entity ID) | `https://api.pulumi.com/login/**acmecorp**/sso/saml/metadata` |
-| Reply URL | `https://api.pulumi.com/login/**acmecorp**/sso/saml/acs` |
-| Relay State | `https://api.pulumi.com/login/**acmecorp**/sso` |
+| Identifier (Entity ID) | `https://api.pulumi.com/login/<acmecorp>/sso/saml/metadata` |
```

Comment:
Could put a big, yellow warning sign on the paragraph above?

We've had reports from some users not replacing `acmecorp` when configuring their SAML SSO settings, which leads to failures logging in. (And could you also make the corresponding warning/message in our Okta, G Suite, and Azure AD SAML guides too?)

Ultimately this is evidently a bigger stumbling block than you'd think.

Reply:
What do you think of http://content-bucket-1ddffb8.s3-website-us-east-1.amazonaws.com/docs/guides/saml/aad/#enter-pulumi-configuration-into-your-azure-ad-application? @cnunciato this was the warning class I came up with previously, but we could update this (or revisit later) if you have some other suggestions.

Reply:
I also added a `required` badge here: http://content-bucket-1ddffb8.s3-website-us-east-1.amazonaws.com/docs/guides/saml/sso/#default-relay-state